docs/python/src/model/model.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import psutil
+
+from PySide2.QtGui import QGuiApplication
+from PySide2.QtQml import QQmlApplicationEngine, qmlRegisterType
+from PySide2.QtCore import QUrl, QTimer, QAbstractListModel
+
+from PySide2.QtCore import QObject, Signal, Slot, Property, Qt
+
+class CpuLoadModel(QAbstractListModel):
+    def __init__(self):
+        QAbstractListModel.__init__(self)
+        
+        self._cpu_count = psutil.cpu_count()
+        self._cpu_load = [0] * self._cpu_count
+        
+        self._updateTimer = QTimer(self)
```

Comment:
the same styling comment apply here, for `_updateTimer`.
Please do consider than when implemented required methods by Qt-classes we keep the camelCase style like for the method `rowCount`

Reply:
Fixed!